jetty-core/jetty-io/src/test/java/org/eclipse/jetty/io/ArrayRetainableByteBufferPoolTest.java

```diff
@@ -350,42 +349,31 @@ public void testExponentialPool() throws IOException
         }
 
         b3.release();
-        b4.getBuffer().limit(b4.getBuffer().capacity() - 2);
-        assertThat(pool.dump(), containsString("]{capacity=4,inuse=3(75%)"));
+        b4.getByteBuffer().limit(b4.getByteBuffer().capacity() - 2);
+        assertThat(pool.dump(), containsString("{capacity=4,inuse=3(75%)"));
     }
 
     @Test
     public void testEndiannessResetOnRelease()
     {
         ArrayRetainableByteBufferPool bufferPool = new ArrayRetainableByteBufferPool();
         RetainableByteBuffer buffer = bufferPool.acquire(10, true);
-        assertThat(buffer.getBuffer().order(), Matchers.is(ByteOrder.BIG_ENDIAN));
-        buffer.getBuffer().order(ByteOrder.LITTLE_ENDIAN);
+        assertThat(buffer.getByteBuffer().order(), Matchers.is(ByteOrder.BIG_ENDIAN));
+        buffer.getByteBuffer().order(ByteOrder.LITTLE_ENDIAN);
         assertThat(buffer.release(), is(true));
-        assertThat(buffer.getBuffer().order(), Matchers.is(ByteOrder.BIG_ENDIAN));
+        assertThat(buffer.getByteBuffer().order(), Matchers.is(ByteOrder.BIG_ENDIAN));
     }
 
     @Test
     public void testLogarithmic()
```

Comment:
Since the logarithmic pool has been replaced with the quadratic one, isn't this redundant with `testQuadraticPool`?